Dockerfile

```diff
@@ -0,0 +1,28 @@
+
+FROM maven:3.8.1-openjdk-17 AS build
+
+
+WORKDIR /app
+
+COPY pom.xml /app/
+COPY src /app/src/
+
+
+RUN mvn clean package -DskipTests
+
+
+FROM openjdk:17-jdk-slim
+
+WORKDIR /app
+
+COPY --from=build /app/target/rihal-0.0.1-SNAPSHOT.jar /app/rihal-0.0.1-SNAPSHOT.jar
+
+EXPOSE 8080
+
+ENV SPRING_DATASOURCE_URL=jdbc:mysql://mysql:3307/usersystem?useSSL=false&serverTimezone=UTC&createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true
```

Comment:
If we change the values directly in the Dockerfile, then we would need to rebuild the Docker image to apply these changes. This is because the environment variables are baked into the image during the build process.